src/rp2_common/hardware_dma/include/hardware/dma.h

```diff
@@ -891,6 +891,18 @@ static inline uint dma_get_timer_dreq(uint timer_num) {
     return DREQ_DMA_TIMER0 + timer_num;
 }
 
+/*! \brief Performs DMA channel cleanup after use
```

Comment:
perhaps the doc could be a bit clearer about what a "cleanup"'s purpose/function is